File: hw1/src/hw1/Company.java
/*
 * Name: Jack Ambery
 * Assignment: hw1
 * Date: 13 September 2022
 * This is the Company class for hw1.
 * 
 */
package hw1;

public class Company {
	
	private Employee[] employees = new Employee[10];
	int position = 0;
	
	public void addEmployee(Employee emp) {
		employees[position] = emp;
		position++;
	}
	
	public int getEmployeeCount() {
		return position;
	}
}

File: c++/hw1/Company.cpp
#include "Company.h"
#include "Employee.h"
#include <iostream>
using namespace std;

Company::Company(vector<Employee> employees_) 
    : employees(employees_) {
}

void Company::addEmployee(Employee e) {
    employees.push_back(e);
}

int Company::getEmployeeCount() {
    cout << employees.size();
}

Company::~Company() {
    cout << "clearing space of the company..." << endl;
}

